test/acceptance/tools/sth.py

```diff
@@ -0,0 +1,290 @@
+        """
+        configuration values
+        """
+        # service
+        if service.find(RANDOM_SERVICE_LENGTH) >= 0:
+            characters = int(service.split(" = ")[1])
+            self.service = general_utils.string_generator(characters, CHARS_ALLOWED)
+        else:
+            self.service = service
+
+        # service_path
+        if service_path.find(RANDOM_SERVICE_PATH_LENGTH) >= 0:
+            characters = int(service_path.split(" = ")[1])
+            self.service_path = "/"+general_utils.string_generator(characters, CHARS_ALLOWED)
+        else:
+            self.service_path = service_path
+        #if self.service_path[:1] == "/": self.service_path = self.service_path[1:]
+
+        # entity_type and entity id
+        self.entity_type = entity_type
+        self.entity_id   = entity_id
+        #attributes
+        self.attributes_number = attributes_number
+        self.attributes_name   = attributes_name
+        self.attribute_type    = attribute_type
+
+    def get_timestamp_remote(self):
+        """
+        return date-time in timestamp from sth server
+        :return float
+        """
+        myfab = FabricSupport(host=self.sth_host, user=self.fabric_user, password=self.fabric_password, cert_file=self.fabric_cert_file, retry=self.fabric_error_retry, hide=True, sudo=self.fabric_sudo)
+        return float(myfab.run("date +%s"))  # get timestamp
+
+    def received_notification(self, attribute_value, metadata_value, content):
+        """
+        notifications
+        :param attribute_value: attribute value
+        :param metadata_value: metadata value (true or false)
+        :param content: xml or json
+        """
+        self.date_time = None
+        self.content = content
+        self.metadata_value = metadata_value
+        metadata_attribute_number = 1
+        notification = Notifications (self.sth_url,tenant=self.service, service_path=self.service_path, content=self.content)
+        if self.metadata_value.lower() == "true":
+            notification.create_metadatas_attribute(metadata_attribute_number, RANDOM, RANDOM, RANDOM)
+        notification.create_attributes (self.attributes_number, self.attributes_name, self.attribute_type, attribute_value)
+        resp = notification.send_notification(self.entity_id, self.entity_type)
+
+        self.date_time = self.get_timestamp_remote()
+        self.attributes = notification.get_attributes()
+        self.attributes_name = notification.get_attributes_name()
+        self.attributes_value =notification.get_attributes_value()
+        self.attributes_number = notification.get_attributes_number()
+        self.attributes_number = int(notification.get_attributes_number())
+        return resp
+
+    def drop_database_in_mongo(self):
+         """
+         delete database and collections in mongo
+         """
+         world.mongo.connect("%s_%s" % (STH_DATABASE_PREFIX, self.service))
```

Comment:
Dont use "world" in a external library of lettuce, pass the instance by argument